### PR TITLE
Missing dependencies added, grafana version fixed to latest stable

### PR DESCRIPTION
**Pull request by alkk**

### CODE REVIEW

**Comment by bbccdd** [comment]
Thanks for the PR. I've merged the changes manually pending maintainers approval.

**Review by bbccdd — APPROVED** [review]
These changes (update grafana to more recent version) indeed work as expected.

**Comment by sweoggy** [comment]
Thanks!

**Pull request merged**
